### DISCUSSION TITLE
Trying to trigger script from Elementor popup

### DISCUSSION THREAD

**Discussion by stevethewebsiteguy** [elementor/elementor#14533] [Developers Community]
I'm having an issue with a script not firing from a popup. I have created a test page here:

https://www.usshingle.com/test/

The popup triggers after 8 seconds. The "get estimate" button on the popup is supposed to trigger an estimate form. I have added a second "get estimate" button on the physical page and it works just fine. Trying to figure out why it won't trigger from the popup. Thanks for any help!

This is the script: <script src="[https://calculator.roofr.com/roofr-calculator.js](https://calculator.roofr.com/roofr-calculator.js)" cid="4235"></script>
And this is the button code: <div class="roofr-calculator"></div>

**Comment by nkoerner93** [comment]
Not sure if I undertand it correclty, but as far as I know you cant have a popuptrigger inside a popup. Cant you create a seperate form page with a popup on it that opens when landing on the page?

**Reply by stevethewebsiteguy** [reply]
Yeah, it's not supported yet but wasn't sure if someone knew of a way to make this happen. Just wanted to have one less step for site visitors. Currently the popup goes to another page with the script which works just fine but the client prefers to have it trigger immediately from the popup. Elementor does plan on adding this in the future but not sure what that actual time frame is. You can have an Elementor popup within an Elementor popup but not a third party script from an Elementor popup.

**Reply by nkoerner93** [reply]
Yeah we've had the same issue on one of our pages and haven't found a solution for it and had to end up with the redirect to another page. Depending on the form its not that bad, because the direct site-view can be more comfortable for some users, especially on mobile devices. Maybe someone else can help you though
